vendor/assets/javascripts/schedule/schedule.js

```diff
@@ -51,6 +51,10 @@ window.Schedule = function (scheduleContainer, options) {
     eventElement.style.borderColor        = BORDER_COLORS[colorIndex];
     eventBackground.style.backgroundColor = BACKGROUND_COLORS[colorIndex];
 
+    if("tooltip" in event) {
```

Comment:
```
if ('tooltip' in event)
  eventElement.title = event.tooltip;
```

We really need to add a linter to enforce coding style... but I'd prefer spaces before parenthesis in control flow and single quotes wherever possible. I could be swung on the option curly brackets though.